docs/create-packages/Prerelease-Packages.md

```diff
@@ -17,15 +17,15 @@ A stable release is one that's considered reliable enough to be used in producti
 
 To support the software release lifecycle, NuGet 1.6 and later allows for the distribution of pre-release packages, where the version number includes a semantic versioning suffix such as `-alpha`, `-beta`, or `-rc`. For more information, see [Package versioning](../reference/package-versioning.md#pre-release-versions).
 
-You can specify such versions in three ways:
+You can specify such versions using one of the following ways:
 
-- `.nuspec` file: include the semantic version suffix in the `version` element:
+- [`nuget.exe`](../tools/nuget-exe-cli-reference.md): include the semantic version suffix in the `.nuspec` file's `version` element:
 
     ```xml
     <version>1.0.1-alpha</version>
     ```
 
-- `.csproj` file: include the semantic version suffix in the `PackageVersion` element:
+- [dotnet CLI](/dotnet/articles/core/tools/index): include the semantic version suffix in the `.csproj` file's  `PackageVersion` element:
```

Comment:
I think we should refrain from using dotnet CLI as a feature descriptor. 

We prefer csproj with further clarification wherever possible.